fluent-tests/src/test/java/com/azure/mgmttest/CompilationTests.java

```diff
@@ -175,4 +179,9 @@ public void testSchemaCleanup() {
     public void testResourceWithWritableName() {
         FirewallRuleInner firewallRuleInner = new FirewallRuleInner();
     }
+
+    public void testNonStringExpandableEnum() throws IOException {
+        ScheduledQueryRuleProperties properties = new ScheduledQueryRuleProperties();
+        String jsonString = properties.toJsonString();
```

Comment:
Why does the model have `toJsonString` method? I didn't recall we had any model with this method?

Reply:
This is provided by JsonSerializable's default implementation: 
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/serialization/azure-json/src/main/java/com/azure/json/JsonSerializable.java#L78-L84

Reply:
I see. 1.2.0?

Reply:
Yes. https://github.com/Azure/azure-sdk-for-java/pull/41253/files#diff-0197788260b7e501fdb9e98100746846527a9a78b1c06dfa6327ec39e93ed9beR78